### PR TITLE
Write missing tests in functions view

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Write missing tests in functions view:
  - for `CodeAndDeps` editors
  - for apollo hooks
  - for `Configuration` tab
  - for `ResourceManagement` panel 

**Related issue(s)**
https://github.com/kyma-project/console/issues/1748

### CODE REVIEW

File: core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ResourceManagement/test/ResourceManagement.test.js [hunk 1]
```diff
@@ -1,47 +1,273 @@
...
+      const saveButton = getByText(saveText);
+      expect(saveButton).not.toBeDisabled();
+      expect(saveButton).toBeInTheDocument();
+      const cancelButton = getByText(BUTTONS.CANCEL);
+      expect(cancelButton).not.toBeDisabled();
+      expect(cancelButton).toBeInTheDocument();
+    });
+  });
+
+  it('show again Edit button after click Cancel', async () => {
```
[the remark this answers is not in the document]
Comment (aerfio) [reply core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ResourceManagement/test/ResourceManagement.test.js:66]: and also this may be some kind of corner case, but with k8s you never know when something blows up:
when you get incorrect values from backend you should still be able to start editing, and then validation should pop in and saving those incorrect values, as far as I remember, we've coded it to work like that (and the cause was that k8s turned cpu values from milicores to microcores or even nanocores :V)

File: core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ResourceManagement/ResourceManagement.js [hunk 2]
```diff
@@ -94,16 +45,17 @@ export function ResourcesManagement({ lambda }) {
     type: UPDATE_TYPE.RESOURCES_AND_REPLICAS,
   });
 
-  const resetFields = () =>
+  function resetFields() {
     Object.entries(defaultValues).forEach(([name, val]) => setValue(name, val));
+  }
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ResourceManagement/ResourceManagement.js:50]: I can revert it XD Sorry :P

File: core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ResourceManagement/test/ResourceManagement.test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ResourceManagement/test/ResourceManagement.test.js:66]: I will add.

File: core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ResourceManagement/test/ResourceManagement.test.js [hunk 3]
```diff
@@ -1,47 +1,273 @@
...
+        RESOURCES_MANAGEMENT_PANEL.ERROR_MESSAGES.MIN_REPLICAS_NON_NEGATIVE,
+    },
+    {
+      name: 'should cannot save when user type non negative max replicas',
+      id: '#maxReplicas',
+      errorMessage:
+        RESOURCES_MANAGEMENT_PANEL.ERROR_MESSAGES.MAX_REPLICAS_NON_NEGATIVE,
+    },
+  ];
+  for (const testCase of replicasTestCases) {
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ResourceManagement/test/ResourceManagement.test.js:105]: I didn't use it specially because it is not legible in the code itself
